docs/tutorial-smart-ai-1.md

```diff
@@ -0,0 +1,208 @@
+(18604, 186040, 0, 0, 1, 0, 0, NULL);
+
+DELETE FROM `waypoint_data` WHERE `id` = 186040;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES
+(186040, 1, -4044.12, -3393.59, 38.1363, NULL, 6000), -- 1s, Talk 0, 5s, Talk 1
+(186040, 2, -4044.21, -3394.23, 38.3905, NULL, 0),
+(186040, 3, -4042.96, -3396.48, 38.3905, NULL, 0),
+(186040, 4, -4041.71, -3397.23, 38.3905, NULL, 0),
+(186040, 5, -4040.8, -3396.88, 38.1447, NULL, 16000), -- 2s, Emote 16, 6s, Talk 2, 6s, Garion Talk 0
+(186040, 6, -4043.67, -3395.27, 38.1634, NULL, 0),
+(186040, 7, -4043.67, -3395.27, 38.1634, 3.87463, 240000);
+```
+
+Breaking down this line by line:
+
+```
+UPDATE `creature` SET `MovementType` = 2 WHERE `id1` = 23600 AND `guid` = 18604;
+```
+
+We're not placing a new creature, just adding behaviour to an existing one, so we will just modify its MovementType, which tells it will use the path set in its creature_addon entry.
+
+```
+DELETE FROM `creature_addon` WHERE (`guid` IN (18604));
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(18604, 186040, 0, 0, 1, 0, 0, NULL);
+```
+
+This is its creature_addon entry. Patrols have, by convention, ids equal to GUID*10. The most common value for bytes2, also known as SheateState, is 1, which represents the melee weapons of the creature. This also contains RP-related items such as lanterns, bottles and such.
+
+```
+DELETE FROM `waypoint_data` WHERE `id` = 186040;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES
+(186040, 1, -4044.12, -3393.59, 38.1363, NULL, 6000), -- 1s, Talk 0, 5s, Talk 1
+(186040, 2, -4044.21, -3394.23, 38.3905, NULL, 0),
+(186040, 3, -4042.96, -3396.48, 38.3905, NULL, 0),
+(186040, 4, -4041.71, -3397.23, 38.3905, NULL, 0),
+(186040, 5, -4040.8, -3396.88, 38.1447, NULL, 16000), -- 2s, Emote 16, 6s, Talk 2, 6s, Garion Talk 0
+(186040, 6, -4043.67, -3395.27, 38.1634, NULL, 0),
+(186040, 7, -4043.67, -3395.27, 38.1634, 3.87463, 180000);
+```
+
+This is the waypoint patrol the creature is assigned to. I've shortened it to not contain columns we'll not use.
+
+The x,y,z values are self-explanatory. An orientation change requires an extra point copied from the one before, otherwise it's null. Delay will dictate the wait time after that waypoint is reached, as such, after the creature finishes the path on point 7, it will wait 3 minutes before starting the patrol again.
+
+I also added a few comments detailing what actions are taken on which waypoints. It's a simplified version of what we'll be writing in the Actionlist.
+
+## Setting up the main events
+
+I use Keira to edit SmartAI, so that's what we'll use in this tutorial.
+
+![image](https://github.com/user-attachments/assets/72296501-d8a0-4c01-a73a-d6a7c0824d22)
```

Comment:
upload the images to /assets/images/ then fetch them like this

```
![image](assets/images/image.png)
```

Note that they won't show when you view the file in the PR but will show when the page is built